osmosis/assetlist.json

```diff
@@ -165,6 +165,50 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/axelar/images/usdc.svg"
       }
     },
+    {
+      "denom_units": [
+        {
+          "denom": "ibc/0B3C3D06228578334B66B57FBFBA4033216CEB8119B27ACDEE18D92DA5B28D43",
+          "exponent": 0,
+          "aliases": [
+            "avalanche-uusdc"
+          ]
+        },
+        {
+          "denom": "avalanche-usdc",
+          "exponent": 6
+        }
+      ],
+      "type_asset": "ics20",
+      "base": "ibc/0B3C3D06228578334B66B57FBFBA4033216CEB8119B27ACDEE18D92DA5B28D43",
+      "name": "Wormhole USDC(Avalanche)",
+      "display": "avalanche-usdc.hole",
+      "symbol": "avaxUsdc.hole",
+      "traces": [
+        {
+          "type": "bridge",
+          "counterparty": {
+            "chain_name": "avalanche",
+            "base_denom": "0xB97EF9Ef8734C71904D8002F8b6Bc66Dd9c48a6E"
+          },
+          "provider": "Wormhole"
+        }
+      ],
+      "images": [
+        {
+          "image_sync": {
+            "chain_name": "osmosis",
+            "base_denom": "avalanche-uusdc"
+          },
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/usdc.hole.png",
+          "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/usdc.hole.svg"
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/osmosis/images/usdc.hole.png",
```

Comment:
Does Wormhole use the same logo for USDC from Ethereum and for USDC from Avalanche?

Reply:
they just use a generic USDC logo, at least in portalbridge.... Do you want to use the same generic logo? I find more explicit a specific one for Avalanche, since the unwrapped liquidity rest there. But you tell me :)

Reply:
I think this is fine for now, but Osmosis will have to create a new image for this asset anyway, and I'll update this once it gets added.

Reply:
alright, thanks!